cli/internal/transformer/transformer.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 const (
-	cqSyncTime     = "_cq_sync_time"
-	cqSourceName   = "_cq_source_name"
-	cqIDColumnName = "_cq_id"
+	cqSyncTime       = "_cq_sync_time"
+	cqSourceName     = "_cq_source_name"
+	cqIDColumnName   = "_cq_id"
+	cqExternalSyncId = "_cq_external_sync_id"
```

Comment:
I get caught up on the name, because this ID is not designed to identify a single sync it is designed to identify a group of syncs so maybe something like `_cq_sync_group_id`?

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/17683/commits/05f5c40aa8fe693119f8d4c94da19e2b7c35e532